src/frontend/src/header.tsx

```diff
@@ -219,6 +212,14 @@ const UserSection = ({ user }: { user: UserType }) => {
                             >
                                 <Journal /> JOURNAL
                             </a>
+                            <a
+                                title="PROFILE"
+                                className="icon_link"
+                                href={`/#/user/${user.id}`}
+                                onClick={hide}
+                            >
+                                <User /> PROFILE
```

Comment:
Shouldn't we have the avatar here? Actually I expected in the header which is removed now.

Reply:
Ok